packages/api-v2/test/student/student.e2e-spec.ts

```diff
@@ -0,0 +1,125 @@
+import { INestApplication } from "@nestjs/common";
+import { Test, TestingModule } from "@nestjs/testing";
+import { Plan } from "../../src/plan/entities/plan.entity";
+import { Student } from "../../src/student/entities/student.entity";
+import * as request from "supertest";
+import { Connection } from "typeorm";
+import { AppModule } from "../../src/app.module";
+
+const testUser = {
+  fullName: "Tester",
+  nuid: "000000000",
+  email: "test-student@gmail.com",
+  password: "1234567890",
+  academicYear: 2019,
+  graduateYear: 2023,
+  catalogYear: 2019,
+  major: "Computer Science",
+};
+
+const testPlan = {
+  name: "Test Student",
+  schedule: {
+    years: [2019, 2020, 2021, 2022],
+    yearMap: {},
+  },
+  major: "Computer Science",
+  coopCycle: "4 year 2 co-ops",
+  concentration: "Artificial Intelligence",
+  catalogYear: 2019,
+  courseWarnings: [],
+  warnings: [],
+};
+
+describe("StudentController (e2e)", () => {
+  let app: INestApplication;
+  let jwtToken: string;
+  let connection: Connection;
+  let uuid: string;
+
+  beforeEach(async () => {
+    const moduleFixture: TestingModule = await Test.createTestingModule({
+      imports: [AppModule],
+    }).compile();
+    app = moduleFixture.createNestApplication();
+
+    await app.init();
+
+    connection = app.get(Connection);
```

Comment:
this appears to be repetitive in all tests, can extract to a function that just does this and returns app